logic/pulsed/sequence_generator_logic.py

```diff
@@ -157,7 +168,18 @@ def on_activate(self):
         # sampling functions from.
         sf_path_list = [os.path.join(get_main_dir(), 'logic', 'pulsed', 'sampling_function_defs')]
         if isinstance(self._sampling_functions_import_path, str):
-            sf_path_list.append(self._sampling_functions_import_path)
+            self._sampling_functions_import_path = [self._sampling_functions_import_path]
```

Comment:
Similar here. Please check for default value (`None`)

Reply:
see above.